test/specs/mountconfigs/fstab.yaml

```diff
@@ -1,2 +1,3 @@
 mountpoints:
-  /: https://adobe.sharepoint.com/sites/TheBlog/Shared%20Documents/theblog?csf=1&e=8Znxth
\ No newline at end of file
+  - path: /
```

Comment:
I don't like the change in the YAML from a map to a list because it makes things more verbose and harder to read.

Additionally, the change to a list is not necessary, as we get the YAML in a data structure that preserves the order, so all we have to do is to provide a good proxy handler for the `/mountpoints$` pattern that picks the correct resolver/decorator.

I can take a stab at this.

Reply:
> I don't like the change in the YAML from a map to a list because it makes things more verbose and harder to read

as explained in #250, we might need to extend the config anyways, e.g. to add an explicit type or to add additional properties, the mount handler might need.